packages/cascara/src/ui/Form/Form.fixture.js

```diff
@@ -157,10 +161,16 @@ export default {
       data={{}}
       dataDisplay={dataDisplay}
       isInitialEditing
+      name='test-form'
     />
   ),
   dataWithDisplay: (
-    <DataWithDisplay actions={actions} data={data} dataDisplay={dataDisplay} />
+    <DataWithDisplay
+      actions={actions}
+      data={data}
+      dataDisplay={dataDisplay}
+      name='test-form'
```

Comment:
Awesome to see that this apparently works! I did not know about this until I saw the notes in the issue opened in RTL.